### PR TITLE
[improve](move-memtable) introduce brpc stream patch to make stream write to socket in background bthread

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

 Introduce brpc stream patch to make stream write to socket in background bthread.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

File: be/src/common/config.h [hunk 1]
```diff
@@ -820,6 +820,8 @@ DECLARE_mDouble(tablet_version_graph_orphan_vertex_ratio);
 DECLARE_Bool(share_delta_writers);
 // timeout for open load stream rpc in ms
 DECLARE_Int64(open_load_stream_timeout_ms);
+// enable write background when using brpc stream
+DECLARE_mBool(enable_write_background);
```
Comment (liaoxin01) [inline be/src/common/config.h:824]:
```suggestion
DECLARE_mBool(enable_brpc_stream_write_background);
```

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**